apps/api/src/app/feature-flags/services/launch-darkly.service.ts

```diff
@@ -17,14 +17,12 @@ export class LaunchDarklyService implements IFeatureFlagsService {
     }
   }
 
-  public async initialize(): Promise<void> {
-    try {
-      await this.client.waitForInitialization();
-      Logger.log('Launch Darkly SDK has been successfully initialized', LOG_CONTEXT);
-    } catch (error) {
-      Logger.error('Launch Darkly SDK has failed when initialized', LOG_CONTEXT, error);
-      throw error;
-    }
+  public async get<T>(key: FeatureFlagKey, context: IFeatureFlagContext, defaultValue: T): Promise<T> {
```

Comment:
Now sure if the following suggestion is manageable 😅.
What do you think we will create a backoff functionality?
1. it could be another SDK of memory storage hardcoded/environment variable/or other.
2. it could be a provider base class that points to memory data.
3. it could be a feature-flag-service backoff if `get` failed.
4. or any other way.

The purpose is to backoff to a predefined flag if the service is not responsive.

The tricky part here is to manage this data. Maybe we can retrieve a JSON from launch darkly and store it as it is. But once again the managing part could be tricky.

Another option could be manual backoff per flag in the code. but this option could be less convenient because we will need to update the code => deploy

If you feel functionality is needed we can create a task for it.